lib/apiservers/engine/backends/archive.go

```diff
@@ -447,7 +442,7 @@ func (wm *ArchiveStreamWriterMap) WriterForAsset(proxy proxy.VicArchiveProxy, ci
 			deviceID = aw.mountPoint.Name
 			store = constants.VolumeStoreName
 		}
-		rawWriter, err := proxy.ArchiveImportWriter(wm.op, store, deviceID, aw.filterSpec)
+		rawWriter, err := proxy.ArchiveImportWriter(wm.op, store, deviceID, aw.filterSpec, wm.errchan)
```

Comment:
The CloseWithErrors() works if one PL call returns before all writes are finished. But the error can happen after all pipewriter finishes, and in that case, we won't catch them with closeWithErrors because there won't be anymore writes.